infra/main.bicep

```diff
@@ -47,6 +47,7 @@ param openAiResourceGroupLocation string
 
 param openAiSkuName string = 'S0'
 
+@secure()
```

Comment:
You will also need to resolve the #919 b/c bicep will give you a warning that you shouldn't output keys.

Reply:
Ah darn. Converting PR to draft.